modules/components/pages/inputs/spicedb_watch.adoc

```diff
@@ -0,0 +1,260 @@
+= spicedb_watch
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+Consumes messages from the https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WatchService.Watch[Watch API^] of a https://authzed.com/docs/spicedb/getting-started/discovering-spicedb[SpiceDB^] instance. This input is useful if you have downstream applications that need to react to real-time changes in data managed by SpiceDB.
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common configuration fields, showing default values
+input:
+  label: ""
+  spicedb_watch:
+    endpoint: grpc.authzed.com:443 # No default (required)
+    bearer_token: ""
+    cache: "" # No default (required)
+```
+--
+Advanced::
++
+--
+```yml
+# All configuration fields, showing default values
+input:
+  label: ""
+  spicedb_watch:
+    endpoint: grpc.authzed.com:443 # No default (required)
+    bearer_token: ""
+    max_receive_message_bytes: 4MB
+    cache: "" # No default (required)
+    cache_key: authzed.com/spicedb/watch/last_zed_token
+    tls:
+      enabled: false
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+```
+--
+======
+
+== Authentication
+
+For this input to authenticate with your SpiceDB instance, you must provide:
+
+- The endpoint of the SpiceDB instance
+- A Bearer token
+
+== Configure a Cache
+
+You must use a xref:configuration:resources.adoc[cache resource] to store the https://authzed.com/docs/spicedb/concepts/consistency#zedtokens[ZedToken^] (ID) of the latest message consumed and acknowledged by this input. Ideally, the cache should persistent across restarts. This means that every time the input is initialized, it starts reading from the newest data updates.
```

Comment:
Is this accurate? As `cache` is a required field, could we give an example?

Reply:
Yes, it's accurate, the cache is mandatory to store the latest consumed ZedToken.
You could add a reference to the resources and cache docs. Something like "to learn how to configure a cache see...":
- resources: https://docs.redpanda.com/redpanda-connect/configuration/resources/
- cache: https://docs.redpanda.com/redpanda-connect/components/caches/about/


The config with a cache could be something like this:
```
input:
  label: ""
  spicedb_watch:
    endpoint: grpc.authzed.com:443 # No default (required)
    bearer_token: ""
    cache: "spicedb_cache"
cache_resources:
  - label: "spicedb_cache"
    redis:
      url: redis://:6379
```

Reply:
I've added the example and updated the associated text.